Readability.js

```diff
@@ -844,18 +844,50 @@ Readability.prototype = {
 
         this._initializeNode(topCandidate);
       } else if (topCandidate) {
+        parentOfTopCandidate = topCandidate.parentNode;
+
+        // Find a better top candidate node if it contains (at least three) nodes which belong to `topCandidates` array
+        // and whose scores are quite closed with current `topCandidate` node.
+        var alternativeCandidateAncestors = [];
+        for (var i = 1; i < topCandidates.length; i++) {
+          if (topCandidates[i].readability.contentScore / topCandidate.readability.contentScore >= 0.75) {
+            alternativeCandidateAncestors.push(this._getNodeAncestors(topCandidates[i]));
+          }
+        }
```

Comment:
Node supports `let` after [version 4.3.2][node-4-3-2]. iOS might support it after [version 10.1][ios-10-1].

I think we could start to use `let` after over 95% (even 100%) of iOS users upgrade to version 10.1+. For now, only [78.1% of iOS users][upgrade-ios-10-1] have upgraded to version 10.1+. What do you think?

[node-4-3-2]: http://node.green/#let
[ios-10-1]: http://caniuse.com/#feat=let
[upgrade-ios-10-1]: https://david-smith.org/iosversionstats